### PR TITLE
Add two showcase from ICLighthouse

### PR DESCRIPTION
**Pull request by elephant-w**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sesi200** [comment]
The CLA bot broke once again... Would you mind going through the flow over here too? https://github.com/dfinity/cla/issues/279

File: showcase.json [hunk 1]
```diff
@@ -1958,5 +1958,46 @@
...
+      "DeFi"
+    ],
+    "usesInternetIdentity": false,
+    "github": "",
+    "twitter": "",
+    "youtube": "",
+    "submittableId": "36483996"
+  },
+  {
+    "name": "BTC integration in ICDex",
```
Comment (sesi200) [inline showcase.json:1983]: ICDex is already in the showcase and this tile seems somewhat empty. In this state I don't think it should be added

Comment (elephant-w) [reply showcase.json:1983]: Hello, what we added was not the ICDex project, but projects related to BTC and ETH integration. These projects are primarily intended for grant applications.

Comment (jennifertrin) [reply showcase.json:1983]: Hello @elephant-w We allow only one listing per core product on the showcase. Would you be open to highlighting that you offer a BTC and ETH integration on the ICLighthouse listing instead?

**Comment by jennifertrin** [comment]
Hello @elephant-w As mentioned above, we allow only one listing per core product on the showcase. If you would like to highlight that you offer a BTC and ETH integration, please update your description in a new PR.

**Pull request closed without merging**